package.json

```
@@ -10,7 +10,7 @@
     "services:down": "docker compose -f infra/compose.yaml down",
     "lint:check": "prettier --check .",
     "lint:fix": "prettier --write .",
-    "test": "jest --runInBand",
+    "test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest --hide next -k -s command-jest 'next dev' 'jest --runInBand'",
```

Minha sugestão é adicionar `npm run services:stop` no fim do comando para encerrar o docker, ficando assim:

```suggestion
    "test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest --hide next -k -s command-jest 'next dev' 'jest --runInBand' && npm run services:stop",
```
Não sei dizer até que ponto isso é relevante para um script de teste mas depois da aula:
https://curso.dev/web/configurar-scripts-servicos
Em que tiveram algumas sugestões  para poupar recursos da máquina encerrando a execução dos containers após finalizar a aplicação com um `CTRL+C`, eu tenho pensado mais nisso e sempre tento encerrar os processos que não serão mais utilizados.